app/docs/0.11.x/admin-api.md

```diff
@@ -1705,53 +1711,6 @@ HTTP 200 OK
 
 ---
 
-### List active targets
-
-Retrieve a list of active targets (targets whose most recent weight is not 0)
-for a given upstream.
-
-<div class="alert alert-warning">
-  <strong>Note:</strong> This endpoint is only available with Kong 0.10.1+
-</div>
-
-### Endpoint
-
-<div class="endpoint get">/upstreams/{name or id}/targets/active</div>
```

Comment:
This endpoint seems to be mis-represented. It needs a trailing slash like so:
```
/upstreams/{name or id}/targets/active/
```

The changes proposed in this PR look healthy to me (making the `created_at` field's meaning explicit - but instead of removing this endpoint, we should probably simply add the trailing slash it needs.

It used not to require a trailing slash, but recent changes in LuaJIT and Lapis made it so that it is now required. If the endpoint was declared without a trailing slash in the Kong codebase, it could - once again - not require a trailing slash. Again, this is a breaking change introduced by recent changes in LuaJIT and Lapis.